SoObjects/SOGo/NSString+Utilities.m

```diff
@@ -492,36 +492,7 @@ - (NSString *) pureEMailAddress
 
 - (NSString *) asQPSubjectString: (NSString *) encoding
 {
-  NSString *qpString, *subjectString;
-  NSData *subjectData, *destSubjectData;
-  NSUInteger length, destLength;
-  unsigned char *destString;
-
-#warning "encoding" parameter is not useful
```

Comment:
This warning is still valid, do you think it is feasible to remove the parameter from the callers?